core/src/test/scala/integration/kafka/api/EndToEndAuthorizationTest.scala

```diff
@@ -60,7 +55,7 @@ import scala.util.{Failure, Success}
   * SaslTestHarness here directly because it extends ZooKeeperTestHarness, and we
   * would end up with ZooKeeperTestHarness twice.
   */
-trait EndToEndAuthorizationTest extends IntegrationTestHarness with SaslSetup {
+abstract class EndToEndAuthorizationTest extends IntegrationTestHarness with SaslSetup {
```

Comment:
I found this was required in order to get JUnit to see the `expected` argument in the `@Test` annotation.